src/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Leaderboard</title>
+</head>
+<body>
+    <h1>Leaderboard</h1>
+    <div class="container">
+        <article class="recent-scores">
+          <div class="refresh-scores">
+            <h2>Recent scores</h2>
+            <input type="submit" id="refresh-button" value="Refresh">
+          </div>
+          <div id="scores-display"></div>
+        </article>
+        <article class="add-scores">
+          <h2>Add your score</h2>
+          <div class="input-scores">
+            <input id="user-name" name="user-name" placeholder="Your name" required />
+            <input id="user-score" name="user-score" placeholder="Your score" required />
+            <input id="user-data-submit" type="submit" value="Submit" />
+          </div>
+        </article>
+    </div>
+</body>
+</html>
```

Comment:
- [ ] There is no need for two index.html files. index.html file should be present only in the dist folder. so I would recommend you remove this one. 😄

Reply:
The /dist folder contains the minimized version of the source code. The code present in the /dist folder is actually the code which is used on production web applications. Along with the minified code, the /dist folder also comprises of all the compiled modules that may or may not be used with other systems.